arch/risc-v/include/types.h

```diff
@@ -78,6 +87,14 @@ typedef __WCHAR_TYPE__     _wchar_t;
 typedef int                _wchar_t;
 #endif
 
+/* Use uintreg_t for register-width integers */
+
+#ifdef CONFIG_ARCH_RV32
```

Comment:
need check CONFIG_ARCH_RV64ILP32 too

Reply:
Seems no need, as `ARCH_RV64ILP32` imples `ARCH_RV64` because it depends on `ARCH_RV64`. It is just a new ABI for `RV64`.